extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcUtils.java

```diff
@@ -242,13 +243,12 @@ private static List<String> convertJsonArrayToList(JsonArray claimValue) {
     static QuarkusSecurityIdentity validateAndCreateIdentity(
             RoutingContext vertxContext, TokenCredential credential,
             TenantConfigContext resolvedContext, JsonObject tokenJson, JsonObject rolesJson, UserInfo userInfo,
-            TokenIntrospection introspectionResult) {
+            TokenIntrospection introspectionResult, Map<String, Object> requestData) {
```

Comment:
I believe the `vertxContext` param should no longer be needed

Reply:
it is needed because this util method is putting the `vertxContext` to the identity it is building.

Reply:
It is just getting a bit confusing, some methods work with RoutingContext, some not, and I'm not sure which code paths are covered and which are not. I'd propose remove `RoutingContext` import all together, prepare the request map in CodeAuthenticationMechanism and pass it at the authentication context attribute, as well as the routing context attribute but only for the code which sets it as a SecurityIdentityAugmentor attribute to work

Reply:
Done.